common/reviews/api/rush-stack-compiler.api.ts

```diff
@@ -0,0 +1,109 @@
+// @beta
+class ApiExtractorRunner extends RushStackCompilerBase<IApiExtractorTaskConfig> {
+  constructor(taskOptions: IApiExtractorTaskConfig, constants: Constants, terminalProvider: ITerminalProvider);
+  // (undocumented)
+  invoke(): Promise<void>;
+}
+
+// @beta
+class CmdRunner<TTaskConfig extends IRushStackCompilerBaseOptions> {
+  constructor(constants: Constants, terminal: Terminal, options: IBaseTaskOptions<TTaskConfig>);
+  // (undocumented)
+  protected _getArgs(): string[];
+  // (undocumented)
+  protected _onClose(code: number, hasErrors: boolean, resolve: () => void, reject: (error: Error) => void): void;
+  // (undocumented)
+  protected _onData(data: Buffer): void;
+  // (undocumented)
+  protected _onError(data: Buffer): void;
+  // (undocumented)
+  runCmd(options: IRunCmdOptions): Promise<void>;
+}
+
+// @beta (undocumented)
```

Comment:
Does this need to be exported at all?  Maybe all we need is the root path.

I don't really want RSC to be an API that anybody uses.  RS or RS-core should fil that role